### PR TITLE
updated main page

### PR DESCRIPTION
**Pull request by deborahokangi**
updated main page and added css files

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -1,14 +1,360 @@
...
-        <a href="/user?id=adam123">User</a>
+<!doctype html>
+<html lang="en" data-bs-theme="auto">
+  <head><script src="../assets/js/color-modes.js"></script>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Hugo 0.118.2">
```
Comment (kubestaff) [inline html/index.html:9]: why hugo?

File: html/index.html [hunk 2]
```diff
@@ -1,14 +1,360 @@
...
+    <title>Album example · Bootstrap v5.3</title>
+
+    <link rel="canonical" href="https://getbootstrap.com/docs/5.3/examples/album/">
+
+    
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3">
+
+<link href="../static/css/bootstrap.min.css" rel="stylesheet">
```
Comment (kubestaff) [inline html/index.html:18]: why ../ is needed?

File: html/index.html [hunk 3]
```diff
@@ -1,14 +1,360 @@
...
+
+      .bd-mode-toggle .dropdown-menu .active .bi {
+        display: block !important;
+      }
+    </style>
+
+    
+  </head>
+  <body>
+    <svg xmlns="http://www.w3.org/2000/svg" class="d-none">
```
Comment (kubestaff) [inline html/index.html:102]: put svg to a separate file with .svg extension

File: html/index.html (hunk 2 above)
Comment (deborahokangi) [reply html/index.html:18]: that is the path to the css file

**Review by kubestaff — APPROVED** [review]

**Pull request merged**